data-products/src/main/scala/org/sunbird/analytics/exhaust/OnDemandExhaustJob.scala

```diff
@@ -180,13 +182,20 @@ trait OnDemandExhaustJob {
       // $COVERAGE-OFF$ Disabling scoverage
       case "s3" =>
         CommonUtil.getS3File(storageConfig.container, "");
+      case "oci" =>
+        CommonUtil.getS3File(storageConfig.container, "");
       case "azure" =>
         CommonUtil.getAzureFile(storageConfig.container, "", storageConfig.accountKey.getOrElse("azure_storage_key"))
       // $COVERAGE-ON$ for case: local
       case _ =>
         storageConfig.fileName
     }
     val objKey = url.replace(filePrefix, "");
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "url" -> url)), INFO)
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "filePrefix" -> filePrefix)), INFO)
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "localPath" -> localPath)), INFO)
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "objKey" -> objKey)), INFO)
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "objKey" -> objKey)), INFO)
```

Comment:
@ddevadat Please avoid multiple logs. Since it is map, all the values can be put it into one log.

Reply:
removed